config/locales/it.yml

```diff
@@ -597,7 +597,7 @@ it:
               
               <hr3>Moderazione</h3>
               <p>
-              Il team di moderazione si riserva di intervenire in caso vengano pubblicati sulla piattaforma contenuti illegali o esplicitamente fuori contesto rispetto ai temi di interesse della comunità. Confidiamo nella capacità della comunità di auto-moderarsi attraverso lo strumento del downvote che invitiamo ad usare con giudizio. La moderazione interverrà inoltre per reprimere comportamenti tossici e ostili verso membri della comunità o terzi, in linea con i principi e le regole esposte nel Code of Conduct.
+              Il team di moderazione si riserva di intervenire in caso vengano pubblicati sulla piattaforma contenuti illegali o esplicitamente fuori contesto rispetto ai temi di interesse della comunità. Confidiamo nella capacità della comunità di auto-moderarsi attraverso lo strumento del downvote che invitiamo ad usare con giudizio. La moderazione interverrà inoltre per reprimere comportamenti tossici e ostili verso membri della comunità o terzi.
```

Comment:
_Il team di moderazione_ -> La nostra squadra di moderatori

Reply:
non si era detto basta traduzioni forzate? Team usato in questo modo è italiano corretto